### ISSUE TITLE
Problem installing guppy part with pip

### ISSUE THREAD

**Issue by ijobling** [codio/boxparts#263]
for both Python2 and Python 3, error shown referencing gcc

See pip log in /ianjobling/pipguppy

**Comment by yaboi** [comment]
Yep, this is the problem I'm having when trying to install it from `pip install -r requirements.txt` or just `pip install guppy` with Python 2.

**Comment by yaboi** [comment]
+1
